examples/udp-listener/udp_listener.bal

```diff
@@ -0,0 +1,35 @@
+// This is the server implementation for the UDP socket.
+import ballerina/io;
+import ballerina/udp;
+import ballerina/log;
+
+// Bind the service to the port.
+// Optionally, you can provide the `remoteHost` and `remotePort` to
+// configure the listener as a connected listener, which only
+// reads and writes to the configured remote host.
+// udp:Listener(48829, remoteHost = "www.remote-clinet.com", remotePort = 8080)
+service on new udp:Listener(48829) {
+
+    // This remote method is invoked once the content is received from the client.
+    // You may replace the onBytes method with onDatagram which reads the data as
+    // readonly & udp:Datagram.
+    remote function onBytes(readonly & byte[] data, udp:Caller caller) 
+            returns (readonly & byte[])|udp:Error? {
+        io:println("Received by listener:", getString(data));
```

Comment:
Add a space after ":". We need to be consistent with the format of the output messages.

Reply:
updated